src/devtools/useAtomDevtools.ts

```diff
@@ -95,3 +95,8 @@ export function useAtomDevtools<Value>(
     }
   }, [anAtom, extension, atomName, value])
 }
+
+export const useAtomDevtools: UseAtomDevtools =
+  process.env.NODE_ENV === 'development' && typeof window !== 'undefined'
```

Comment:
Any comments on the condition? I'm not sure what's good.

Reply:
Again, no problems with using `type` here for the function. Having the `type` and then the `useAtomDevtoolsImpl` implementation isn't a pattern I normally use in my own projects, but I can see where this would be helpful for a library like Jotai.

Reply:
With regards to disabling in production, there should maybe be an additional parameter on the hook (i.e. `useAtomDevtools(..., enableInProduction = false)`) to override this behavior if for some reason one wishes to enable the devtools in a production build.

Reply:
OK, that was my concern. Unfortunately, the additional parameter doesn't work because of the rules of hooks. I'll revert the change.